Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1204,6 +1204,101 @@ def setup_(self, step, stepName, stepDict, k, properties):
+    },
+    suffix = 'Patatrack_HCALOnlyAlpaka_Profiling',
+    offset = 0.431,
+)
+
+# HCAL-PF Only workflow running HCAL local reco on GPU and PF with Alpaka with DQM and Validation
+# - HLT on GPU
+# - HCAL-only reconstruction using Alpaka with DQM and Validation
+upgradeWFs['PatatrackHCALOnlyAlpakaValidation'] = PatatrackWorkflow(
+    digi = {
+        # the HLT menu is already set up for using GPUs if available and if the "gpu" modifier is enabled
+        '--procModifiers': 'gpu'
+    },
+    reco = {
+        '-s': 'RAW2DIGI:RawToDigi_hcalOnly,RECO:reconstruction_hcalOnly,VALIDATION:@hcalOnlyValidation,DQM:@hcalOnly+@hcal2Only',
+        '--procModifiers': 'alpaka,gpu'
+    },
+    harvest = {
+        '-s': 'HARVESTING:@hcalOnlyValidation'
+    },
+    suffix = 'Patatrack_HCALOnlyAlpaka_Validation',
+    offset = 0.432,
+)
+
+# HCAL-PF Only workflow running HCAL local reco on CPU and PF with Alpaka with cluster level-validation
+# - HLT on CPU
+# - HCAL-only reconstruction using Alpaka with DQM and Validation for Alpaka vs CPU comparisons
+upgradeWFs['PatatrackHCALOnlyCPUandAlpakaValidation'] = PatatrackWorkflow(
+    digi = {
+        # the HLT menu is already set up for using GPUs if available and if the "gpu" modifier is enabled
+    },
+    reco = {
+        '-s': 'RAW2DIGI:RawToDigi_hcalOnly,RECO:reconstruction_hcalOnly,VALIDATION:@hcalOnlyValidation,DQM:@hcalOnly+@hcal2Only',
+        '--procModifiers': 'alpaka,alpakaValidationParticleFlow'
+    },
+    harvest = {
+        '-s': 'HARVESTING:@hcalOnlyValidation'
+    },
+    suffix = 'Patatrack_HCALOnlyCPUandAlpaka_Validation',
+    offset = 0.433,
+)
+
+# HCAL-PF Only workflow running HCAL local reco on GPU and PF with Alpaka with cluster level-validation
+# - HLT on GPU (optional)
+# - HCAL-only reconstruction using GPU and Alpaka with DQM and Validation for Alpaka vs CPU comparisons
+upgradeWFs['PatatrackHCALOnlyGPUandAlpakaValidation'] = PatatrackWorkflow(
+    digi = {
+        # the HLT menu is already set up for using GPUs if available and if the "gpu" modifier is enabled
+        '--procModifiers': 'gpu'
+    },
+    reco = {
+        '-s': 'RAW2DIGI:RawToDigi_hcalOnly,RECO:reconstruction_hcalOnly,VALIDATION:@hcalOnlyValidation,DQM:@hcalOnly+@hcal2Only',
+        '--procModifiers': 'alpaka,alpakaValidationParticleFlow,gpu'
+    },
+    harvest = {
+        '-s': 'HARVESTING:@hcalOnlyValidation'
+    },
+    suffix = 'Patatrack_HCALOnlyGPUandAlpaka_Validation',
+    offset = 0.434,
+)
+
+# HCAL-PF Only workflow running HCAL local reco on GPU and PF with Alpaka (GPU) with cluster level-validation
+# - HLT on GPU (required)
+# - HCAL-only reconstruction using GPU and Alpaka GPU backend with DQM and Validation for Alpaka vs CPU comparisons
+upgradeWFs['PatatrackHCALOnlyAlpakaGPUValidation'] = PatatrackWorkflow(
+    digi = {
+        # the HLT menu is already set up for using GPUs if available and if the "gpu" modifier is enabled
+        '--accelerators': 'gpu-nvidia',
```

Comment:
Please use `gpu-*` rather than `gu-nvidia`, with Alpaka we want to run these workflows also on non-NVIDIA GPUs.